pkgs/development/python-modules/cffi/default.nix

```diff
@@ -32,7 +32,7 @@ if isPyPy then null else buildPythonPackage rec {
   NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.cc.isClang
     "-Wno-unused-command-line-argument -Wno-unreachable-code";
 
-  doCheck = !stdenv.hostPlatform.isMusl && !stdenv.isDarwin; # TODO: Investigate
+  doCheck = false; #!stdenv.hostPlatform.isMusl && !stdenv.isDarwin; # TODO: Investigate
```

Comment:
This will need to be fixed before we merge. I can have a look at that.

Reply:
A lot of `AssertionError: assert 'linux' == 'win32'` and `OSError: ctypes.util.find_library() did not manage to locate a library called 'm'`

Reply:
What is the status on this part?

Reply:
The errors sadly didn't magically disappear... I will try to make the tests pass tomorrow.

Reply:
I fixed the tests on 3.7 and 3.8 by backporting a fix of ctypes.util.find_library.